CMakeLists.txt

```diff
@@ -269,13 +260,12 @@ add_library(ut INTERFACE)
 target_include_directories(ut SYSTEM INTERFACE "${CMAKE_CURRENT_SOURCE_DIR}/thirdparty/ut")
 
 # cURL
-if(NOT DISABLE_CURL)
+if(ENABLE_CURL)
```

Comment:
It doesn't strictly belong here, but I wouldn't be against hardcoding these to true. The usability of minifi is very limited without curl and openssl.

Reply:
Good idea, i've created a ticket for this. https://issues.apache.org/jira/browse/MINIFICPP-2251